### PR TITLE
fix(issues): 2024-120-2 bis

### PR DESCRIPTION
**Pull request by percebus**
# Issues

- https://github.com/percebus/azure-secure-networking-for-devs/issues/26
- https://github.com/percebus/azure-secure-networking-for-devs/issues/25
- https://github.com/percebus/azure-secure-networking-for-devs/issues/17
- https://github.com/percebus/azure-secure-networking-for-devs/issues/31
- https://github.com/percebus/azure-secure-networking-for-devs/issues/50

### CODE REVIEW

File: docs/tutorial/01/hub.md [hunk 1]
```diff
@@ -293,14 +293,18 @@ These are standard, to ensure connectivity with a minimum level of security on r
 
 Your resources should look like this.-
 
-![snapshot](../../../assets/img/azure/solution/vnets/hub/snapshots/02.png)
+![snapshot](../../../assets/img/azure/solution/vnets/hub/snapshots/01.png)
```
Comment (percebus) [inline docs/tutorial/01/hub.md:296]: https://github.com/percebus/azure-secure-networking-for-devs/issues/25

**Review by percebus — COMMENTED** [review]
Ping of Death

File: assets/img/azure/solution/vnets/hub/rt/routes/exit-vnet-thru-fw.png
Comment (percebus) [inline assets/img/azure/solution/vnets/hub/rt/routes/exit-vnet-thru-fw.png:1]: https://github.com/percebus/azure-secure-networking-for-devs/issues/17

**Review by timmyreilly — APPROVED** [review]
Thank you JC!

**Pull request merged**